app/views/shared/_flash_msg.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render 'global_alerts/alerts' if Settings.global_alert %>
+<div class="flash_messages">
```

Comment:
Perhaps you don't want to show this div if it's content is empty. This could get flagged as an accessibility issue.

Reply:
Doesn't that rule only apply if the element or role has a required owned element (which I don't think is true of a generic div)? https://www.w3.org/TR/wai-aria/#mustContain

Reply:
Yeah, you're right.